internal/clientconn/listener.go

```diff
@@ -159,11 +160,9 @@ func (l *Listener) Addr() net.Addr {
 // Describe implements prometheus.Collector.
 func (l *Listener) Describe(ch chan<- *prometheus.Desc) {
 	l.metrics.Describe(ch)
-	l.handlersMetrics.Describe(ch)
```

Comment:
That's why we no longer collect request/response metrics, but we should collect them

Reply:
added as conn metrics gathering